config/rbac/role.yaml

```diff
@@ -26,3 +26,23 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - velero.io
+  resources:
+  - serverstatusrequests
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Given the controller isn't set up for a `watch` verb, do we want to have it listed in the RBAC permissions?

Reply:
I will be removing this file in a separate PR, we don't need it.

Reply:
Created a ticket to track clean ups related to the migrations: https://github.com/vmware-tanzu/velero/issues/2884